scripts/lib/CIME/test_scheduler.py

```diff
@@ -880,7 +881,9 @@ def run_tests(self, wait=False,
         # Setup cs files
         self._setup_cs_files()
 
+        GenericXML.DISABLE_CACHING = True
```

Comment:
Shouldn't this be done as a function call for good form?

Reply:
Maybe, it should only be temporary though.

Reply:
okay